src/styles/main.scss

```diff
@@ -62,22 +63,68 @@ body {
 }
 
 .addBtn {
-  margin-left: 1rem;
-  background-color: #271c6c;
-  color: #e1ebfd;
+  margin-left: 15px;
+  background-color: $btn-bcg;
+  color: $white-color;
   border-radius: 50%;
-  border: 2px solid #e1ebfd;
-  font-size: 1.5rem;
-  width: 3.2rem;
-  height: 3.2rem;
+  border: 2px solid $white-color;
+  font-size: 24px;
+  width: 50px;
+  height: 50px;
   cursor: pointer;
-  box-shadow: 2px 4px 10px #271c6c;
+  box-shadow: 2px 4px 10px $btn-bcg;
 
- @include displayFlex;
+ @include displayFlex(0, center, center);
 }
 
 .addBtn:focus {
   outline: none;
 }
 
+.displayTodos {
+  margin-top: 48px;
+  display: flex;
+  @include displayFlex(column, 0, center);
+}
+
+.displayTodos .buttons {
+  margin-bottom: 32px;
+}
+
+.displayTodos .buttons button {
+  padding: 8px 20px;
+  border-radius: 20px;
+  cursor: pointer;
+  border: none;
+}
```

Comment:
use nesting in styles:
.displayTodos {
  margin-top: 48px;
...
  
  .buttons {
  margin-bottom: 32px;
}

...
}